### PR TITLE
Cleanup navigation/evan

### PR DESCRIPTION
**Pull request by technabors**
Updating Nav and Global layer docs with new images / specs / documentation. Addresses SURF 171 / 172 / 173

### CODE REVIEW

File: components/tables.html [hunk 1]
```diff
@@ -84,6 +84,23 @@
         <li>Make sure that any row that is sortable has the sort hover states</li>
         <li>It’s helpful to include an information icon with a tooltip that explains the sorting function on sortable rows</li>
       </ul>
+      <ul>
```
Comment (nimbinatus) [inline components/tables.html]: Any reason for ending one list only to immediately start a new one? If this list was supposed to go under that last bullet, take out `</ul>` on line 86 and move the `</li>` from the end of line 85 to a new line after line 103.

File: layout/global-layer.html [hunk 2]
```diff
@@ -26,22 +26,36 @@ <h2 id="when">When to Use</h2>
...
+  <p>The eyebrow is more commonly referred to as the Pilot API Nav Header within the application frame because it provides for easy switching between product lines and platform level services. Platform level services are any services that exist across multiple product lines. Ex: Account Settings, Notifications, Ticketing, etc. For details specs please see the <a href="{{site.baseurl}}/layout/navigation#eyebrow">navigation page</a></p>
 
   <div class="row">
   	 <div class="col-md-4">
       <a class="anchor" name="top-nav"></a>
         <ul>
           <li>Extends across the top of the entire Helix-UI and is always visible</li>
-          <li>Contains links to platform level services and the <a href="#global">global dashboard</a></li>
+          <li>Contains links to platform level services, account management, product picker and the <a href="#global">global dashboard</a></li>
+          <li>Composed of 5 breakpoints that adapt to the users device and window size</a></li>
```
Comment (nimbinatus) [inline layout/global-layer.html]: Take the `</li>` at the end of this line, and place it on a new line after line 44. [Reference](https://www.w3.org/wiki/HTML_lists#Nesting_lists)

File: layout/navigation.html [hunk 3]
```diff
@@ -18,11 +18,97 @@
...
   <p>The primary navigation is used for the main sections of the application or for dropdown categories that organize sections into higher level groups.</p>
 
+  <div class="row" id="side-nav">
+    <div class="col-md-4">
+      <h2>Left Navigation</h2>
+      <p>The Left Navigation component is used on all web applications that desire to implement a highly compliant version of the Helix Design Language.</p>
+      <ul>
+        <li>Can extend to fit any navigation IA via scrolling when the list of navigational items exceeds the screen view-height.</li>
+        <li>Preferred implementation includes only primary and secondary navigational links.</li>
+          <ul>Note: A less preferred implementation that includes <a href="{{page.url}}#tertiary"> tertiary navigational</a> elements is also provided.</ul>
```
Comment (nimbinatus) [inline layout/navigation.html]: Should `<ul>` and `</ul>` here be `<li>` and `</li>`?

File: layout/navigation.html [hunk 4]
```diff
@@ -18,11 +18,97 @@
...
+    </div>
+    <div class="col-md-7 col-md-offset-1">
+      <figure>
+        <div class="image bg-light border">
+          <img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/layout/navigation-left-spacing-horizontal.svg">
+        </div>
+        <figcaption>Containing div spacing specs</figcaption>
+      </figure>
+    </div>
+    <div class="col-md-4">
```
Comment (nimbinatus) [inline layout/navigation.html]: Is this line missing the `<div class="row">` before it?

File: layout/navigation.html [hunk 5]
```diff
@@ -54,47 +142,77 @@
   <div class="row">
     <div class="col-md-4">
       <h2>Top Nav Spacing</h2>
-      <p>Horizontal spacing includes 20px left margin
-      <br>Each menu item button has 20px left and right padding
-      <br>Each menu item button has 20px padding in between</p>
+      <ul>
+        <li>Horizontal spacing includes 20px left margin</li>
+        <li>Each menu item button has 20px left and right padding</li>
```
Comment (nimbinatus) [inline layout/navigation.html]: See note re:nested lists above.

File: layout/navigation.html [hunk 6]
```diff
@@ -112,63 +230,113 @@
...
     </div>
-  </div>
-
-  <div class="row">
-    <div class="col-md-4">
-      <h2>Left Navigation Color and Typeface</h2>
-      <p>Roboto Regular 16px
-      <br>Gray 400 #D8D8D8
-      <br>Title Case</p>
+    <h4>Eyebrow (left side) Dropdowns</h4>
```
Comment (nimbinatus) [inline layout/navigation.html]: Missing the row-ending `</div>` here.

File: layout/navigation.html [hunk 7]
```diff
@@ -18,11 +18,97 @@
...
+    </div>
+    <div class="col-md-7 col-md-offset-1">
+      <figure>
+        <div class="image bg-light border">
+          <img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/layout/navigation-left-spacing-vertical.svg">
+        </div>
+        <figcaption>Vertical spacing between navigational items</figcaption>
+      </figure>
+    </div>
+    <div class="col-md-4">
```
Comment (nimbinatus) [inline layout/navigation.html]: Same here; seems to be missing the end of one row and the beginning of the next.

File: layout/navigation.html [hunk 8]
```diff
@@ -54,47 +142,77 @@
...
     </div>
     <div class="col-md-7 col-md-offset-1">
       <figure>
         <div class="image bg-light border">
           <img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/layout/navigation-top-spacing-vertical.svg">
         </div>
+        <figcaption>Vertical padding for primary navigational items</figcaption>
       </figure>
+    </div>
+    <div class="col-md-4">
```
Comment (nimbinatus) [inline layout/navigation.html]: Again, seems to be missing the end of one row and the start of the next.

File: layout/navigation.html [hunk 9]
```diff
@@ -112,63 +230,113 @@
...
     <div class="col-md-7 col-md-offset-1">
       <figure>
         <div class="image bg-light border">
-          <img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/layout/navigation-left-spacing-vertical.svg">
+          <img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/layout/global-layer/eyebrow-right.svg">
         </div>
+        <figcaption>Eyebrow (rightside) construction specs</figcaption>
       </figure>
+    </div>
+  <h4>Eyebrow (right side) Dropdowns</h4>
```
Comment (nimbinatus) [inline layout/navigation.html]: Missing the row-ending `</div>` here.

File: layout/navigation.html [hunk 10]
```diff
@@ -179,7 +347,7 @@
     <div class="image bg-light border">
       <img src="http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/layout/navigation-eyebrow-encore.svg">
     </div>
-  </figure>
+  </figure>-->
```
Comment (nimbinatus) [inline layout/navigation.html]: Row-ending div missing.

File: components/tables.html (hunk 1 above)
Comment (technabors) [inline components/tables.html]: Thanks, fixed.

File: layout/navigation.html (hunk 3 above)
Comment (technabors) [inline layout/navigation.html]: We don't have a defined style to nest a note under an li currently, so i nested a ul in the interim to ensure it indented as supporting information to the li above . I'll make a task to create a new note style from things such as this in the future.

Comment (nimbinatus) [inline layout/navigation.html]: :+1: I probably have a set of styles squirreled away from the IX sites for that sort of note that we can use...

**Comment by technabors** [comment]
Merging post review with Laura.

---------
Laura Santamaria [4:17 PM] 
good deal :slightly_smiling_face: So I can’t merge for you or anything, but you’ve got a stamp of approval from me.
---------

**Pull request merged**